src/app/shared/functions/modular.functions.ts

```diff
@@ -197,3 +197,20 @@ export function isMobile() {
         return false;
     }
 }
+
+export const scrollToFirstError = (selector, element) => {
+    let errorElementFound;
+    const checkErrorElementExist = setInterval(() => {
```

Comment:
@Nishanthh Same question here- What's is the use of the setInterval here?

Reply:
@avikt couldn't find the error element instantly, handled with afterviewchecked